.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
```

Comment:
I haven't used travis before, but the time has come. Are you sure of java?) project is written in kotlin.

Reply:
>Are you sure of java?) project is written in kotlin.

But for the JVM ;-) I suppose you do not plan to make this project backend-agnostic like kotlinx.serialization, but if you do - travis config is not set in stone